content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,198 @@ the product page, you can see the following sections:
+because some delay is possible due to caching and other propagation overhead.
 {{< /tip >}}
 
-## Cleanup
-
-*   Remove Istio authorization policy configuration:
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="remove_istio_authorization_policy.sh" >}}
-
-    Alternatively, you can delete all `ServiceRole` and `ServiceRoleBinding` resources by running the following commands:
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="remove_istio_authorization_policy_alternative.sh" >}}
-
-*   Disable Istio authorization:
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="disabling_istio_authorization.sh" >}}
+## Configure access control for workloads using HTTP traffic
+
+Using Istio, you can easily setup access control for {{< gloss "workload" >}}workloads{{< /gloss >}}
+in your mesh. This task shows you how to set up access control using Istio authorization.
+You will start with a simple `deny-all` policy that rejects all requests to the workload,
+and then grant more access to the workload gradually and incrementally.
+
+1. Run the following command to create a policy `deny-all`, the policy has no rule so it
+   cannot grant permission for any traffic. In other words, it denies all:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: deny-all
+      namespace: default
+    spec:
+      {}
+    EOF
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+    You should see `"RBAC: access denied"`. The error shows that the configured `deny-all` policy
+    is working as intended, and Istio doesn't have any rules that allow any access to
+    workloads in the mesh.
+
+    With Istio, you can easily configure namespace-level access control. You can configure how could workloads
+    from a namespace access workloads in another namespace.
+
+1. Run the following command to create a `bookinfo-viewer` policy that allows workloads in the `default`
+   and `istio-system` namespaces to access workloads in the `default` namespace with `GET` method:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: "security.istio.io/v1beta1"
+    kind: "AuthorizationPolicy"
+    metadata:
+      name: "bookinfo-viewer"
+      namespace: default
+    spec:
+      rules:
+      - from:
+        - source:
+            namespaces: ["default", "istio-system"]
+        to:
+        - operation:
+            methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+    Point your browser at Bookinfo's `productpage` (`http://$GATEWAY_URL/productpage`).
+    You should see the "Bookinfo Sample" page, with the "Book Details" section in the lower left part
+    and the "Book Reviews" section in the lower right part.
+
+1. Remove the policy with the following command before continuing:
```

Comment:
Why do you ask readers to remove the policy? Going one step further, why are you applying the policy in the first place? Are you just trying to showcase namespace-level access control?

Reply:
Yes, this is to showcase the namespace-level access control. We need to remove it otherwise the following workload-level policies doesn't make sense because the permission had already been granted in the namespace-level policy.

Reply:
I would suggest you don't show the namespace-level access control here if it is not needed to complete the flow.
Another option would be to complete the flow using namespace-level access control, would that be doable?